src/app/app.component.html

```diff
@@ -1,6 +1,9 @@
 <div class="mainBlocksContainer">
-   <app-header></app-header>
-   <router-outlet></router-outlet>
+   <app-header [logout]="logout" [isUserAuthenticated]="isUserAuthenticated" [userName]="userName"></app-header>
+   <app-login-page [userName]="'NameForLogin'" [login]="login" *ngIf="!isUserAuthenticated; else userIsAuthenticated"></app-login-page>
```

Comment:
Better get this in login component, it isn't necessary in parent component.

Reply:
Actually in this task I have only this app-login-page component (class name is LoginPageComponent) and also in task requirements has been said that: 
"You don’t have to make routing. At this step we don’t know what routing is. So please use familiar construction below:

Place <login-page></login-page> into app component."